### PR TITLE
setup cob46

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

File: cob_bringup/robots/cob4-6.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		</include>
+		<include file="$(find cob_bringup)/tools/teleop.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="version" value="v1"/>
+		</include>
+		<include file="$(find cob_script_server)/launch/script_server.launch" />
+		<!--include file="$(find cob_bringup)/tools/base_collision_observer.launch" >
+			<arg name="robot" value="$(arg robot)" />
+		</include-->
+		<include file="$(find cob_bringup)/drivers/light.launch">
```
Comment (floweisshardt) [inline cob_bringup/robots/cob4-6.xml]: we only have LEDs for cob4-3. cob4-4 to cob4-6 will be shipped without LEDs. Please remove.

File: cob_controller_configuration_gazebo/launch/robots/default_controllers_cob4-6.launch [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	<include file="$(find cob_controller_configuration_gazebo)/launch/components/base_controller_cob4_new.launch">
+		<arg name="robot" value="$(arg robot)"/>
+	</include>
+		
+	<include file="$(find cob_bringup)/drivers/relayboard.launch">
+		<arg name="sim" value="true"/>
+	</include>
+
+	<!-- start additional components -->
+	<include file="$(find cob_bringup)/tools/base_collision_observer.launch" >
```
Comment (floweisshardt) [inline cob_controller_configuration_gazebo/launch/robots/default_controllers_cob4-6.launch]: we should comment this out as for the real bringup. If not done, please comment for cob4-1 and cob4-2 too.

File: cob_default_robot_config/cob4-6/base_configurations.yaml [hunk 3]
```diff
@@ -0,0 +1 @@
+service_ns: /base_controller
```
Comment (floweisshardt) [inline cob_default_robot_config/cob4-6/base_configurations.yaml]: cob4-6 should follow the new namespace structure which is

``` base/driver```
```

File: cob_hardware_config/cob4-6/config/base_controller.yaml [hunk 4]
```diff
@@ -0,0 +1,111 @@
+## joint_names
+joint_names: [fl_caster_rotation_joint, fl_caster_r_wheel_joint, bl_caster_rotation_joint, bl_caster_r_wheel_joint, br_caster_rotation_joint, br_caster_r_wheel_joint]
```
Comment (floweisshardt) [inline cob_hardware_config/cob4-6/config/base_controller.yaml]: for the 3-wheel bases we should rename the joints into

```
fl_*
fr_*
b_*
```

there is no more `bl_*` and `br_*`

Comment (fmessmer) [inline cob_hardware_config/cob4-6/config/base_controller.yaml]: with the current undercarriage_control, this renaming is not possible...

**Comment by floweisshardt** [comment]
visual review: OK

**Pull request merged**